latex-cheatsheet.md

```diff
@@ -0,0 +1,281 @@
+</div>
+versus
+<div class="latexexample">
+$\text{Fubini's theorem gives conditions under which}$
+
+$$\int_{X\times Y} f(x,y)\,\text{d}(x,y) = \int_X\left(\int_Y f(x,y)\,\text{d}y\right)\,\text{d}x$$
+
+</div>
+
+### Large operators
+Mathematics (and therefore $\LaTeX$) contains many large operators, such as $\sum$, $\int$, and $\prod$. Other symbols that act like large operators are $\lim$, $\sup$, $\limsup$, etc. Common between all these symbols is that they are most often seen with expressions placed below and/or above them. Thankfully, this is easy to achieve in $\LaTeX$. The most common way to achieve this is to use the subscript `_` and superscript `^` characters; for example,
+```latex
+\int_a^b
+```
+is rendered in inline mode as
+<div class="latexexample">
+$\int_a^b$
+</div>
+and in display mode as
+<div class="latexexample">
+$$\int_a^b$$
+</div>
+A common mistake people make is in trying to render more complex expressions. For example, you may try to write a sum from $n=1$ to $5$ with `\sum_n=1^5`. This, however, renders as:
+<div class="latexexample">
+$$\sum_n=1^5$$
+</div>
+Yikes. This is where braces enter the picture. Braces, `{` and `}`, are used to group symbols and expressions. The correct way to render what we want is instead `\sum_{n=1}^5`, giving
+<div class="latexexample">
+$$\sum_{n=1}^5$$
+</div>
+A special case of "large operator" is the fraction. Fractions are rendered with their own syntax; namely, `\frac{a}{b}`, which renders in inline mode as
+<div class="latexexample">
+$\frac{a}{b}$
+</div>
+and in display mode as
+<div class="latexexample">
+$$\frac{a}{b}$$
+</div>
+
+### Parentheses
+$\LaTeX$ does not automatically (at least, not by default), scale parentheses to match the vertical height of the enclosed expression. For example, `(\frac{a}{b})` is rendered as
+<div class="latexexample">
+$$(\frac{a}{b})$$
+</div>
+Thankfully, there is an easy means by which to scale parentheses (as well as braces, brackets, and other symbols); specifically, prefix the left symbol with `\left` and the right symbol with `\right`. So, rewriting our example from above, we obtain `\left( \frac{a}{b} \right)`, which is rendered as
+<div class="latexexample">
+$$\left( \frac{a}{b} \right)$$
+</div>
+Note that there is also a `\middle` command that is most often used for matching the height of the vertical bar glyph `|`. An example of this being used may be
+```tex
+\left\{ x \middle\mid x \leq \frac{1}{2} \right\}
+```
+rendered in display mode as
+<!-- BUG For some reason the vertical bar here is not rendering properly -->
+<div class="latexexample">
+$$\left\{ x \middle\lvert x \leq \frac{1}{2} \right\}$$
+</div>
+
+## Using $\LaTeX$ on the server
+The <a class="mention">@TeXiT</a> bot can be used anywhere on the server to render mathematics using $\LaTeX$. To use it, simply write a normal message with any latex expressions that you want to be rendered enclosed in either `$ ... $` or `$$ ... $$`.
```

Comment:
Technically it's anything that contains balanced `$ $`, `$$ $$`, `\( \)` or `\[ \]`. It's a little odd that above you recommend `\(` over `$` but here you do not.